client/src/components/dashboard/ModuleDropdown.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+import './ModuleDropdown.css'
+
+
+const ModuleDropdown = () => {
```

Comment:
Hi Joe, your frontend part looks fine now.

Reply:
But in the future, please provide a meaningful description with a screenshot of your changes